package/MDAnalysis/analysis/encore/clustering/cluster.py

```diff
@@ -45,7 +45,7 @@
 
 
 def cluster(ensembles,
-            method = ClusteringMethod.AffinityPropagationNative(),
+            method=None,
```

Comment:
I'm not sure, I don't really like this because `method=None` is very uninformative. An option would be to allow both `ClusteringMethod`s and strings as inputs, and have a dictionary mapping them? So that the default would become `mathod="AffinityPropagationNative"`, which is informative. But maybe this only adds complexity for not much, so I'd be interested to hear other opinions. (The same applies to the other similar changes.)

Reply:
On my side, I'm "ok" with it for now. It looks like `method` is also documented to accept a list of possible `encore.ClusteringMethod`s, so if we tried to get fancy with allowing strings from which we did i.e., a `getattr` or similar, the internal verification logic may get annoying for checking single string vs. list of string vs. list of appropriate objects.

`ruff` suggests `perform the call within the function, or read the default from a module-level singleton variable`, which doesn't really resolve the matter here I don't think. For the other cases where it is just a mutable object in the arguments, the resolution is much clearer and `ruff` says `Replace with `None`; initialize within function`.

Reply:
Good point on getting to fancy. The logic might get hairy indeed. It could be hidden into a function, but I'm Ok in leaving as-is then.